minimal.css

```diff
@@ -44,26 +50,48 @@ code {
   margin: 1rem;
 }
 
-/* avatar css */
+/* avatar */
 
 .avatar {
   border-radius: 50%;
 }
 
 .avatar-sm {
-    width: 3.125rem; 
-    /* 50px */
-    height: 3.125rem;
+  width: 3.125rem;
+  height: 3.125rem;
 }
 
 .avatar-md {
-    width: 4.688rem;
-    /* 75px */
-    height: 4.688rem;
+  width: 4.688rem;
+  height: 4.688rem;
 }
 
 .avatar-lg {
-    width: 6.25rem;
-    /* 100px */
-    height: 6.25rem;
-}
\ No newline at end of file
+  width: 6.25rem;
+  height: 6.25rem;
+}
+
+/* alert  */
+
+.alert {
+  display: flex;
+  justify-content: center;
+  gap: 0.5rem;
+  width: 25rem;
+  padding: 1rem;
+  border-radius: 0.3rem;
+  color: var(--white);
```

Comment:
You can write CSS variables as var(--white-color).

Reply:
Okay thanks

Reply:
Implemented